src/django/api/templates/mail/approved_facility_claim_contributor_notice_body.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>
+            An OAR facility on one of your facility lists has been claimed
+        </title>
+    </head>
+    <body>
+        <p>
+            Hi,
+        </p>
+        <p>
+            You're receiving this email because we approved a facility claim
+            for a facility appearing on one of your facility lists.
+        </p>
+        <p>
+            The facility is:
+        </p>
+        <ul>
+            <li>
+                Facility: {{ facility_name }}, {{ facility_address }}
```

Comment:
Consider including the full facility country name in these notifications.

Reply:
Added this in 0d3d6a29a1fa3bbfdafd95b0d042eaa5e1c58b2a and also added the full country name to the previous set of facility claim emails.